crowns.lic

```diff
@@ -117,27 +117,21 @@ class Dice
   end
 
   def check_prize
-    if @trash_items.any? { |item| Regexp.new(item) =~ right_hand }
-      bput("put my #{righthand} in bucket", 'You put', 'You drop')
-    else
-      case bput("stow #{righthand}", 'You put', 'to fit in the', 'What were', 'Stow what?')
-      when 'to fit in the'
-        message("*** Item is too big to fit in your stow container! ***")
-        exit
-      end
-      pause
-    end
-    # Check left hand as well. On lag may have items in both hands.
-    if @trash_items.any? { |item| Regexp.new(item) =~ left_hand }
-      bput("put my #{lefthand} in bucket", 'You put', 'You drop')
-    else
-      case bput("stow #{lefthand}", 'You put', 'to fit in the', 'What were', 'Stow what?')
-      when 'to fit in the'
-        message("*** Item is too big to fit in your stow container! ***")
-        exit
+    [left_hand, right_hand]
+      .compact
+      .each do |in_hand|
+        if @trash_items.any? { |item| Regexp.new(item) =~ in_hand }
+          bput("put my #{in_hand} in bucket", 'You put', 'You drop')
+        else
+          case bput("stow #{in_hand}", 'You put', 'to fit in the', 'What were', 'Stow what?')
```

Comment:
💡 @Dartellum this may be a good spot to configuration for _which_ container someone wants their loot to be stowed in in case their default container isn't appropriate. They could specify it in `dice_loot_container`. Thoughts?